extensions/rich_text_components/Image/Image.py

```diff
@@ -47,9 +48,19 @@ class Image(base.BaseRichTextComponent):
         'default_value': '',
     }, {
         'name': 'alt',
-        'description': 'Alternative text (for screen readers)',
+        'description': (
+            'Briefly explain this image to a visually impaired '
+            'learner'),
         'schema': {
             'type': 'unicode',
+            'validators': [{
+                'id': 'is_nonempty',
+            }],
+            'ui_config': {
+                'placeholder': (
+                    'Description of Image (Example : "George Frideric, '
```

Comment:
@kirankonduru Been mulling it over all day, and I'd also recommend removing the quotation marks from around the example. They may give the wrong impression that the description should be in quotes.